### PR TITLE
New Onboarding: show design previews in landscape

### PR DESCRIPTION
**Pull request by lsl**
#### Changes proposed in this Pull Request

- Adds & enables a landscape design preview option in new onboarding
- Enables mshots design previews in all environments
- includes tweak for backing off mshots refreshes the longer it takes
- includes a scroll effect on mousehover for landscape previews

#### Testing instructions

* Check signup displays mshot images with landscape preview http://calypso.localhost:3000/new/design
* Check localized signup displays translated design preview content http://calypso.localhost:3000/new/design/ja
* Check anchor.fm signup flow displays mshot images http://calypso.localhost:3000/new/?anchor_podcast=22b6608

Landscape views:
Screenshot_2021-01-22 WordPress com(4)

Hover effect:
landscape-hover

Relates #49097

### CODE REVIEW

**Comment by ramonjd** [comment]
@lsl I love the hover effect. ❤️ 

On https://hash-445ac0a51ce6c5de81139649cfdfb324bc29023a.calypso.live/new/design?flags=gutenboarding/landscape-preview,gutenboarding/mshot-preview the thumbnails are loading fairly quickly. Emulating slow 3g is what you'd expect for any image-based page IMHO.

It also takes care of the vertical weirdness (the giant hand) we always saw in Rockfield. 

What do you reckon @ollierozdarz @simison ? Would this be acceptable to flick on for the experiment?

**Comment by ollierozdarz** [comment]
@ramonjd I'm happy with it! Let's see what @simison thinks and hopefully we can 🛳  it.

**Comment by lsl** [comment]
Tidied this commit up a bit:

- `?flags=gutenboarding/landscape-previews` will be enabled in production with this deploy (this includes the scroll over effect on both mshots and non mshots previews)
- `?flags=gutenboarding/mshot-previews` will be enabled in development only with this deploy (I want to get systems signoff before we roll this out)

Landscape previews work with both image types but the speed will be slightly different because of the change in image size between mshot/non-mshot.

@ollierozdarz reduced transition time to 1s, feels a lot snappier now, I couldn't get it any faster than that without it stuttering on my machine.

Mshots: https://hash-445ac0a51ce6c5de81139649cfdfb324bc29023a.calypso.live/new/design?flags=gutenboarding/landscape-preview,gutenboarding/mshot-preview

vs no mshots: https://hash-445ac0a51ce6c5de81139649cfdfb324bc29023a.calypso.live/new/design?flags=gutenboarding/landscape-preview

**Review by simison — APPROVED** [review]
Looks great, nice work here @lsl and @ollierozdarz !

Feel free to enable mshots in production if you feel it works well. We'll then get a bit of stress-test before it goes to more traffic with wider testing, and get a bit of internal testing before we roll non-English test out again.

(Didn't code review, leaving that to others)

**Comment by simison** [comment]
One follow-up ask after the merge — let's please ensure screenshots stay fairly big on all screen-sizes. Now you can see how small they become between number of columns:

https://user-images.githubusercontent.com/87168/105838025-3283b880-5fd8-11eb-95a1-66e5221d0c9a.mov

Here are two 3 column versions just before it jumps to 2 columns and previews become bigger again:

Screenshot 2021-01-26 at 13 09 31

Screenshot 2021-01-26 at 13 09 44

This is similar to how it's in production so not a blocker for merge. Feels more prominent on landscape versions though?

**Comment by ollierozdarz** [comment]
> One follow-up ask after the merge — let's please ensure screenshots stay fairly big on all screen-sizes. Now you can see how small they become between number of columns:

Agree 100%, @simison! I think bigger cards, less internal margin, would be the ideal. Even breaking down to the 2-col view earlier as opposed to the 3-col view shrinking tiny then snapping to 2-col.

**Comment by lsl** [comment]
Added image sizing to #49097, had a go at this today but couldn't get it to where I was happy with it. Will circle back with more time.

File: client/landing/gutenboarding/components/mshots-image/index.tsx
Comment (andrewserong) [inline client/landing/gutenboarding/components/mshots-image/index.tsx:74]: I didn't catch this in the last PR, but clever way of detecting the loading image!

File: client/landing/gutenboarding/components/mshots-image/index.tsx
Comment (andrewserong) [inline client/landing/gutenboarding/components/mshots-image/index.tsx:78]: I like that this gradually spaces out the polling gap, a nice easing off.

**Review by andrewserong — APPROVED** [review]
This tests nicely for me across Safari, Edge, Chrome and FF on Mac Desktop, and tested on Safari and Chrome on an iPhone 8. Code change looks good to me — tiniest of tiny nits, I think we're sometimes using mshot singular vs mshots plural, which could trip someone up eventually while trying to change something, but not at all a priority to change in this PR.

Thanks for seeing this through! It's looking really nice.

**Pull request merged**
